scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/service/ScheduleManagerTest.java

```diff
@@ -138,7 +136,7 @@ private void assertCreateAndFindAllSchedules(int noOfSpecificDateSchedules) {
 		assertSpecificSchedulesFoundEquals(noOfSpecificDateSchedules, foundSpecificSchedules);
 
 		// reset all records for next test.
-		afterTest();
+		removeAllRecoredsFromDatabase();
```

Comment:
It was removed as part of commit with comment "Fixed wrong test cases like calling 'afterTest'.".